_config.yml

```diff
@@ -63,4 +63,39 @@ defaults:
       path: "products"
     values:
       layout: product
+      releaseCycleColumn: true
+      releaseDateColumn: false
+      discontinuedColumn: false
+      activesupportColumn: false
+      eolColumn: true
+      columns:
```

Comment:
It does not look like `columns` has to be overridable. So similar to `colorClass`, I think  `columns` should be in its own variable.

Reply:
I did this has a hack to get access to page.columns in Ruby code easily (otherwise, need the `site` variable) for now. Plan to fix this.